dns/octodns-docker/Dockerfile

```diff
@@ -10,5 +10,7 @@ RUN pip install -r /requirements.txt && \
     echo "freezing deps, please make sure requirements.txt matches" && \
     pip freeze | tee /newrequirements.txt && \
     diff /requirements.txt /newrequirements.txt
-
+# RUN pip install ipython ipdb
```

Comment:
Why are these here but commented?  Do we need them or not?

Reply:
We don't need them, but they help with debugging. I turned them off/on several times.
I'll go ahead and remove them.